Dhaval adx data connections

dhavalbhattmsft opened the pull request:
Updated to add:
1. Simplified ADX onboarding doc updates 
2. A short blurb on ADX connection.

dhavalbhattmsft committed:
Add files via upload

dhavalbhattmsft committed:
Delete ADX Data Connections.png

dhavalbhattmsft committed:
Add files via upload

ADX data connection configuration

dhavalbhattmsft committed:
Add files via upload

Grant access to AAD app

dhavalbhattmsft committed:
Update data-connection-adx.md

dhavalbhattmsft committed:
Update index.md

dhavalbhattmsft committed:
Update index.md

dhavalbhattmsft committed:
Configure-ADX-Connection

oh-noey commented on playfab-docs/features/data/dataconnections/index.md:
```diff
@@ -27,6 +27,10 @@ You can get started with data connections for event ingestion using PlayFab’s
 The data is readily available in Parquet blob format. Parquet is a column-oriented data file format designed for efficient data storage and retrieval. It provides efficient data compression and encoding schemes with enhanced performance to handle complex data in bulk, resulting in low latency, higher throughput, and low cost of data storage. It's designed to be a common interchange format for bot ...
 
+### ADX Data Connection
+Data Connections allows you to export to Azure Data Explorer for near-real time ingestion and distribution of your data. ADX, Azure Data Explorer, provides lightning-fast data exploration capabilities, robust analytics, and the ability to handle large volumes of diverse data in near real-time, enabling you to gain invaluable insights and make informed decisions swiftly and efficiently. 
```
We might want to avoid using the term "ADX" in documentation. From prior conversation I believe the data explorer team had recommended against it.

colton-manville replied:
Thomas had mentioned that we were not allowed to use the term ADX in documentation when he had first created these docs

dhavalbhattmsft replied:
removed reference to ADX and added fully acronym.

oh-noey commented on playfab-docs/features/data/dataconnections/index.md:
```diff
@@ -27,6 +27,10 @@ You can get started with data connections for event ingestion using PlayFab’s
 
 Data connections is the replacement for the Event Export and Export to S3 features that currently PlayFab supports. Data connections allow for faster export in preferred formats for offline processing. You get to bring your storage account for data to be ingested and distributed. That means you get to manage and control the encryption-at-rest policy, the lifetime management policy, and network ac ...
 
+### Blob Data Connection
```
Is this header correct? This section seems to apply to data connections in general.

dhavalbhattmsft replied:
Correct.

dhavalbhattmsft committed:
Update index.md

dhavalbhattmsft committed:
Update index.md

updating to fully address ADX acronym.

joannaleecy commented on playfab-docs/features/data/dataconnections/index.md:
```diff
@@ -27,6 +27,10 @@ You can get started with data connections for event ingestion using PlayFab’s
 The data is readily available in Parquet blob format. Parquet is a column-oriented data file format designed for efficient data storage and retrieval. It provides efficient data compression and encoding schemes with enhanced performance to handle complex data in bulk, resulting in low latency, higher throughput, and low cost of data storage. It's designed to be a common interchange format for bot ...
 
+### Azure Data Explorer Data Connection
+Data Connections allows you to export to Azure Data Explorer for near-real time ingestion and distribution of your data. Azure Data Explorer provides lightning-fast data exploration capabilities, robust analytics, and the ability to handle large volumes of diverse data in near real-time, enabling you to gain invaluable insights and make informed decisions swiftly and efficiently. 
```
@dhavalbhattmsft note that you're contributing to the public repo.

Would be great to quantify "large volumes of diverse data" and "lightning-fast". It might mean different things to different people.

dhavalbhattmsft replied:
@joannaleecy  thank you for your input - I've quantified those details.

dhavalbhattmsft committed:
Update index.md

quantified data explorer querying capacity.

joannaleecy commented on playfab-docs/features/data/dataconnections/data-connection-adx.md:
```diff
@@ -55,16 +55,14 @@ Once your Azure AD app is listed as a “Database Admin”, you can move on to c
 | Cluster URI  | The ADX cluster URI.|
```
Pls add some instructions to help developers get the ADX cluster URI.

joannaleecy commented on playfab-docs/features/data/dataconnections/data-connection-adx.md:
```diff
@@ -55,16 +55,14 @@ Once your Azure AD app is listed as a “Database Admin”, you can move on to c
+### Step 3: Ensure that you have granted the Azure Active Directory app access to your Azure Data Explorer cluster before proceeding. 
 
-Return to your Azure AD App’s “App Registration” to create a client secret. You can follow the steps here to create a new client secret for your app: [How to Create a New Application Secret](/azure/active-directory/develop/howto-create-service-principal-portal#option-2-create-a-new-application-secret) 
+Follow the commands as shown in the setup window once you set up your Data connection. 
```
It would be better to list what they need to do briefly in the commands, instead of relying on an image. It's also more accessible.
Also, couldn't tell what I needed to do based on the image.

joannaleecy replied:
I'm expecting a bullet list of things i need to configure or set when i'm in the Azure AD app.

So perhaps start with:

Return to your Azure AD App (add a URL). Navigate to the App Registration page.
ETC.

Be as clear as you can be.

joannaleecy commented on playfab-docs/features/data/dataconnections/data-connection-adx.md:
```diff
@@ -55,16 +55,14 @@ Once your Azure AD app is listed as a “Database Admin”, you can move on to c
 > [!Note]
 > We recommend using a table name that does not exist. If you use an existing table and the schema is not what PlayFab is expecting, then the schema will be modified. 
 
-### Step 3: Create and Share a Client Secret 
```
General comments for the page:

To make it easier to follow.

1) Create a new ADX resource in Azure
2) Link the newly created ADX to PlayFab using Game Manager
3) Show me how to use the connection to get something going :)

joannaleecy replied:
Use "select" instead of "click"

dhavalbhattmsft replied:
Regarding the commands - There is a 'copy` command in the setup window, so if I spell out the commands in the set up instructions, it would be more cumbersome to follow along - especially because these commands are uniquely generated based on the information entered in the previous fields. The commands include the 'keys' and database name etc. so it's less work just to copy paste them. I've updated docs to inform the reader of the availability of 'copy' icon.

dhavalbhattmsft committed:
Update data-connection-adx.md

based on Joanna's feedback around clarity of organization and Cluster URI field details, as well as changing 'clicks' to 'select'
